CHANGELOG.md

```diff
@@ -34,6 +68,8 @@ Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
 - Added ajax subscription submission.
 - Initiative folder and files for Initiative pages
 - Added custom template for FOIA faqs page
+- Added `setup.sh` script for bootstrapping the project.
```

Comment:
Uh oh, why's this an entry in `Unreleased` (https://github.com/cfpb/cfgov-refresh/pull/606/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR19) and in a release?